api/envoy/config/tap/v3/common.proto

```diff
@@ -101,6 +107,30 @@ message HttpHeadersMatch {
   repeated route.v3.HeaderMatcher headers = 1;
 }
 
+// HTTP generic body match configuration.
+// List of text strings and hex strings to be located in HTTP body.
+// All specified strings must be found in the HTTP body for positive match.
+// The search may be limited to specified number of bytes from the body start.
+message HttpGenericBodyMatch {
+  message GenericTextMatch {
+    oneof rule {
+      option (validate.required) = true;
+
+      // Text string to be located in HTTP body.
+      string string_match = 1;
+
+      // Base64 encoded bytes sequence to be located in HTTP body.
```

Comment:
Nit: update comment, no longer base64 encoded.

Other than this, LGTM and ready to approve API.

Reply:
Corrected the comment as indicated.